Package.swift

```diff
@@ -29,10 +29,11 @@ let package = Package(
         )
     ],
     dependencies: [
-        .package(url: "https://github.com/IBM-Swift/Kitura.git", .upToNextMinor(from: "2.1.0")),
-        .package(url: "https://github.com/IBM-Swift/BlueCryptor.git", .upToNextMinor(from: "0.8.0")),
```

Comment:
In semantic versioning the `0.x` versions are reserved for development and breaking changes can be made at any time. Usually projects will use the minor version for breaking changes and patch version for all other changes though. So in this case (until BlueCryptor hits version 1) I think `.upToNextMinor()` is appropriate here.